include/fluent-bit/flb_log.h

```diff
@@ -232,11 +232,18 @@ static inline int flb_log_suppress_check(int log_suppress_interval, const char *
 int flb_log_worker_init(struct flb_worker *worker);
 int flb_log_worker_destroy(struct flb_worker *worker);
 int flb_errno_print(int errnum, const char *file, int line);
+int flb_WSAGetLastError_print(int errnum, const char *file, int line);
 
 #ifdef __FLB_FILENAME__
 #define flb_errno() flb_errno_print(errno, __FLB_FILENAME__, __LINE__)
+#ifdef WIN32
+#define flb_WSAGetLastError() flb_WSAGetLastError_print(WSAGetLastError(), __FLB_FILENAME__, __LINE__)
```

Comment:
Could you please change these names to follow the coding style?

Reply:
I think this function should be renamed, otherwise the current name suggests it would return the error code rather than printing it.

Reply:
I'd appreciate it if you named it `flb_wsa_get_last_error`. 

I agree with you that `flb_errno` presents the same issue in terms of its name but as you might imagine changing its name would involve a boat load of changes which even if minimal would be super noisy so it might not be the right time for that.

What do you think of having both `flb_errno` and `flb_wsa_get_last_error_print` return the error code in addition to printing it just as a first step?

I understand that this would basically eat the error code that the function itself could generate (if error conditions were handled) which could make this a step in the wrong direction so let me know what you think about that.

The "critical" thing to me is changing those names from camel case to snake case.

Reply:
Although no callers will currently use that return value, I don't see any actual harm in simply returning the `errnum` value. Did that along with the rename in [429bbe1](https://github.com/fluent/fluent-bit/pull/10017/commits/429bbe1a0828c848ad19e2b4ba4ffa15c183efce)